code/game/turfs/floor_attackby.dm

```diff
@@ -203,7 +203,7 @@
 						H.shoveling = TRUE
 						visible_message("<span class = 'notice'>[user] starts to shovel dirt into a pile.</span>", "<span class = 'notice'>You start to shovel dirt into a pile.</span>")
 						playsound(src,'sound/effects/shovelling.ogg',100,1)
-						if (do_after(user, (80/(H.getStatCoeff("strength"))/SH.usespeed)))
+						if (do_after(user, (40/(H.getStatCoeff("strength"))/SH.usespeed)))
 							visible_message("<span class = 'notice'>[user] shovels dirt into a pile.</span>", "<span class = 'notice'>You shovel dirt into a pile.</span>")
```

Comment:
change it to 60, we'll make it just a little easier

Reply:
Sounds good.